shell/src/main/java/tachyon/shell/command/CatCommand.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.shell.command;
+
+import java.io.IOException;
+
+import tachyon.TachyonURI;
+import tachyon.client.TachyonStorageType;
+import tachyon.client.file.FileInStream;
+import tachyon.client.file.TachyonFile;
+import tachyon.client.file.TachyonFileSystem;
+import tachyon.client.file.options.InStreamOptions;
+import tachyon.conf.TachyonConf;
+import tachyon.exception.ExceptionMessage;
+import tachyon.exception.TachyonException;
+import tachyon.thrift.FileInfo;
+
+/**
+ * Prints the file's contents to the console.
+ */
+public class CatCommand extends WithWildCardPathCommand {
+
+  public CatCommand(TachyonConf conf, TachyonFileSystem tfs) {
+    super(conf, tfs);
+  }
+
+  @Override
+  public String getCommandName() {
```

Comment:
Could this move the the superclass and "cat" be passed into the constructor? We do it like this in `ClientBase`/`MasterClientBase`